Flick/Controllers/SettingsViewController.swift

```diff
@@ -67,7 +67,9 @@ class SettingsViewController: UIViewController {
     }
 
     @objc func logout() {
-        LoginManager().logOut()
+        if user.socialIdTokenType == "facebook" {
+            LoginManager().logOut()
```

Comment:
what do we do if apple?

Reply:
apple doesn't make a log out functionality. How an user would really log out is getting rid of the app from settings